src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/TypeImportCompletionItem.cs

```diff
@@ -62,7 +61,6 @@ public static CompletionItem CreateAttributeItemWithoutSuffix(CompletionItem att
 
             return CompletionItem.Create(
                  displayText: attributeNameWithoutSuffix,
-                 filterText: attributeNameWithoutSuffix,
```

Comment:
was this found through tests, a manual audit, something else?

Reply:
Manual audit.